Disaster Recovery/SafeHaven 4/Create-a-SafeHaven-Cluster.md

```diff
@@ -0,0 +1,74 @@
+{{{
+  "title": "Create a SafeHaven Cluster",
+  "date": "05-08-2017",
+  "author": "Sharon Wang",
+  "attachments": [],
+  "contentIsHTML": false
+}}}
+
+### Article Overview
+
+This article explains how to create a SafeHaven Cluster.
+
+### Requirements
+
+1) Pre-deployed CMS server.
+2) Network access to the CMS.
+
+### Assumptions
+
+This article assumes that the user has already deployed the CMS server in CenturyLink Cloud.
+
+To create CMS please refer to [Deploy CMS and SRN in CenturyLink Cloud](Deploy-CMS-and-SRN-in-CenturyLink-Cloud.md)
+
+### SafeHaven Cluster Installation
+
+**NOTE**: Please refer to the latest Safehaven Release Notes to download the  latest **GUI Package**
+
+Begin by downloading the latest SafeHaven Console(GUI) from the [GUI Package](https://download.safehaven.ctl.io/SH-4.0.2/SafeHavenConsole-4.0.2.zip) download link under the **Download Links** section of the [SafeHaven 4.0.2 Release Notes](safehaven-4.0.2-release.md).
+
+Launch the Console by clicking on its icon. Select **Create New Cluster**.
+
+![Upgrade](../../images/SH4.0/Cluster/08.png)
+
+Accept the **End User License Agreement** and click **Next**.
+
+![Upgrade](../../images/SH4.0/Cluster/02.png)
+
+Enter the **Customer Name** and the **License key** provided to you by your CenturyLink Onboarding Engineer.
+
+![Upgrade](../../images/SH4.0/Cluster/03.png)
+
+Fill in the following fields:
+1) CMS hostname in the **Node Name**

```

Comment:
Change numbering from `1)` to `1.`
This will make this KB consistent with the rest of the KBs.